### DISCUSSION TITLE
Close repository on oss.sonatype.org Nexus fail because of invalid SHA256 SHA512 hashes

### DISCUSSION THREAD

**Discussion by kagkarlsson** [jreleaser/jreleaser#1378] [Q&A] [answered]
Hi,

First off, kudos for an impressive project! I am trying to use it to automate the release-process for my open-source project, [db-scheduler](https://github.com/kagkarlsson/db-scheduler).

I am struggling with errors in Nexus during `close`.  See screenshot below. Nexus is complaining about `INVALID SHA-256` and `INVALID SHA-512`. It seems these are not generated by the `deploy-plugin`, but by jreleaser. I have spot-checked some and their values **appear to be correct hashes** of the artifacts' content. Do anyone know what might cause these? Could it be a `https://oss.sonatype.org/` setting or something, like enable support for those hashes or something? (My sonatype account is fairly old, registered 2015)

My previous releases does not have these hashes (manual `mvn deploy`), is there a way to skip generating these two hashes?

<img width="1517" alt="Screenshot 2023-05-02 at 15 21 52" src="https://user-images.githubusercontent.com/172572/235679633-65ce89ac-9d2b-4b3e-87fc-80c176297497.png">

**Comment by aalmiray** [comment, marked as answer]
At the moment there's no option to skip `.sha256` and `.sha512` but you can skip checksum verification altogether by setting `checksums: false` on the nexus2 instance. This will instruct JReleaser to avoid checking and generating missing checksums. Fortunately Maven generates `.sha1` and `.md5` by default which are the minimum expected by Maven Central. Gradle OTOH generates `.md5`, `.sha1`, `.sha256`, and `.sha512`.

You must upgrade to JReleaser `1.6.0` to be able to use this setting as it was recently added. https://github.com/jreleaser/jreleaser/issues/1290

**Reply by kagkarlsson** [reply]
Thank you for the quick reply. `checksums: false` worked 👍  (avoiding the problem for now at least)

**Reply by aalmiray** [reply]
I've seen this problem once before a few weeks ago. Rebuilding all artifacts made the problem go away, but in your case you had them already at the remote staged repository.
